src/transpiler.ts

```diff
@@ -77,6 +77,7 @@ function applyPasses(ast: AST, options: TranspilationOptions): AST {
     ['Ufr', UsingForResolver],
     ['Na', NamedArgsRemover],
     ['Gp', PublicStateVarsGetterGenerator],
+    ['Ech', ExternalContractHandler],
```

Comment:
I think it would probably make most sense to put this after InheritanceInliner, so that any calls to base class functions don't cause interface generation

Reply:
Position Changed!!